packages/turbo/src/BridgeComponent.ts

```diff
@@ -1,48 +1,44 @@
 import { Component } from 'react';
 import type { EmitterSubscription } from 'react-native';
 import type {
-  VisitableViewModule,
   StradaMessage,
   StradaMessages,
   StradaComponentProps,
 } from './types';
-import { getNativeModule, registerMessageEventListener } from './common';
 
-const RNVisitableViewModule = getNativeModule<VisitableViewModule>(
-  'RNVisitableViewModule'
-);
-
-const registerStradaMessageEventListener = (component: BridgeComponent) =>
-  registerMessageEventListener(component.name, (e: object) => {
-    const message = e as StradaMessage;
-    if (message.data.metadata.url !== component.url) {
-      return;
-    }
-    component.previousMessages[message.event] = message;
-    component.onReceive(message);
-  });
+const stradaMessageListener = (component: BridgeComponent) => (e: object) => {
```

Comment:
any reasons why this isn't class method inside `BridgeComponent` ? then you would have component available as `this`

Reply:
https://github.com/software-mansion-labs/react-native-turbo-demo/pull/55#discussion_r1397029729